python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -30,9 +30,11 @@ def create_test(self, name, unit, technique=None, test_id=None):
         raise Exception(res.text)
 
     @verify_credentials
-    def update_test(self, test_id, name=None, unit=None, technique=None):
+    def update_test(self, test_id, name=None, unit=None, technique=None, crowdstrike_expected_outcome: EDRResponse = None):
         """ Update a test """
         body = dict()
+        if crowdstrike_expected_outcome:
+            body[['expected']] = dict(crowdstrike=crowdstrike_expected_outcome.value)
```

Comment:
`[[`? There some cool Python I don't know here?

Reply:
if so, i dont know it either ha